EssentialsDiscord/src/main/java/net/essentialsx/api/v2/services/discord/InteractionCommandArgument.java

```diff
@@ -0,0 +1,57 @@
+package net.essentialsx.api.v2.services.discord;
+
+/**
+ * Represents an argument for a command to be shown to the discord client.
+ */
+public class InteractionCommandArgument {
+    private final String name;
+    private final String description;
+    private final InteractionCommandArgumentType type;
+    private final boolean required;
+
+    /**
+     * Builds a command argument.
+     * @param name        The name of the argument to be shown to the discord client.
+     * @param description A brief description of the argument to be shown to the discord client.
+     * @param type        The type of argument.
+     * @param required    Whether or not the argument is required in order to send the command in the discord client.
```

Comment:
```suggestion
     * @param name        The name of the argument to be shown to the Discord client.
     * @param description A brief description of the argument to be shown to the Discord client.
     * @param type        The type of argument.
     * @param required    Whether or not the argument is required in order to send the command in the Discord client.
```
we Do a little trolling